src/Nethermind/Nethermind.Consensus.AuRa/Json/AbiParameterConverter.cs

```diff
@@ -22,7 +22,7 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
-namespace Nethermind.Serialization.Json.Abi
+namespace Nethermind.Consensus.AuRa.Json
```

Comment:
Why move to Aura? This is generic.

Reply:
when you move the contract code out it will be naturally together